Makefile

```
@@ -0,0 +1,4 @@
+init:
+	pip install -U pip
+	pip install -e .
+	pip install -r requirements.txt
```

사소하지만..  requirements 를 설치하고 custom package를 설치하는게 순서상 맞는 것 같습니다!
```suggestion
	pip install -r requirements.txt
	pip install -e .
```

---

오 그렇네요! 반영하였습니다 :)